### ISSUE TITLE
comment removed from the end of a yaml object

### ISSUE THREAD

**Issue by georgettica** [mikefarah/yq#1217]
**Describe the bug**

when I run `yq -P -i file.yaml` the last comment line is removed

Version of yq: 4.25.1
Operating system: mac
Installed via: homebrew

**Input Yaml**
data1.yml:
```yaml
test:
# this comment will be removed
```

OR

data1.yml:
```yaml
test: {}
# this comment will be removed
```

**Command**
The command you ran:
```
yq -P -i data1.yml
```

**Actual behavior**

```yaml
test:
```

**Expected behavior**

```yaml
test:
# this comment will be removed
```

**Additional context**
didn't test in much, but also setting

```yaml
test: hello
# this comment will be removed
```
does not remove the comment

**Comment by mikefarah** [comment]
Hey I was able to replicate this issue - will fix in the next release.

**Comment by mikefarah** [comment]
Fixed in 4.25.2
